Home.js

```diff
@@ -1,35 +1,47 @@
+import PropTypes from 'prop-types'
 import React, { Component } from 'react'
-import { UserList, Splash } from './components'
 
-import { fetchUsers, fetchProducts } from './helpers/actions'
+import { Splash, UserList } from './components'
+import { fetchProducts, fetchUsers } from './helpers/actions'
 
 export default class Home extends Component {
-  static navigationOptions = ({ navigation }) => {
+  static navigationOptions({ navigation }) {
     if (navigation.getParam('isLoading')) {
       return { header: null }
     }
 
     return { title: 'Flipermercado' }
   }
 
-  isLoading = () => this.props.navigation.getParam('isLoading')
-
   async componentWillMount() {
+    const { props } = this
```

Comment:
Eu não gostei muito desse destructuring. Não fica melhor só o prop/state que vamos usar no contexto?

```suggestion
    const { navigation } = this.props
```

Reply:
Eu também acho! (O @castilh0s também)